src/intl/messages.js

```diff
@@ -824,6 +860,18 @@ const messageDescriptors = {
     description: 'Manage vGPU dialog Display on switch off`',
   },
 
+  vmManageGpuBodyDriverParams: {
+    id: 'vm.gpu.body.driver.properties',
+    defaultMessage: 'Optional driver parameters',
+    description: 'Manage vGPU dialog Optional driver parameters label`',
+  },
+
+  vmManageGpuBodyDriverParamsHelperText: {
+    id: 'vm.gpu.body.driver.properties.helper.text',
+    defaultMessage: 'Available since 4.7 compatibility version',
```

Comment:
The driver parameters are allowed to be set only for NVidia GPUs. I'm not sure whether we support vGPUs for other vendors but in case we do, the limitation should be mentioned here.

Reply:
The only place I found in the documentation that is related to vGPU is titled `SETTING UP AN NVIDIA GPU FOR A VIRTUAL MACHINE IN RED HAT VIRTUALIZATION` so I think that's fine